.github/workflows/ci-test.yml

```diff
@@ -15,9 +15,8 @@ jobs:
     steps:
     - uses: actions/checkout@v3
     - uses: ./.github/actions/install-go-and-dependencies
-    - run: make ci
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - run: make ci-test
     - uses: shogo82148/actions-goveralls@v1
+      if: env.K8S_VERSION == '1.24.x'
```

Comment:
Why restrict this?

Reply:
Because otherwise it's going to do it as many times (5 times), while this needs to happen onyly once

Reply:
We're going to need to constantly maintain this whenever we change Kubernetes versions.

Reply:
We only need to change it if we get rid of `1.24.x` I'll have to figure out how to access the dimensions from here and reference the last one instead of this to avoid that.